Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
-    {{3402, 0x0000}, {"Splat", Game::Superchargers, Element::Magic}},
-    {{3402, 0x4502}, {"Power Blue Splat", Game::Superchargers, Element::Magic}},
-    {{3406, 0x0000}, {"Stormblade", Game::Superchargers, Element::Air}},
-    {{3406, 0x4502}, {"Dark Stormblade", Game::Superchargers, Element::Air}},
-    {{3411, 0x0000}, {"Smash Hit", Game::Superchargers, Element::Earth}},
-    {{3411, 0x4502}, {"Steel Plated Smash Hit", Game::Superchargers, Element::Earth}},
-    {{3412, 0x0000}, {"Spitfire", Game::Superchargers, Element::Fire}},
-    {{3412, 0x4502}, {"Dark Spitfire", Game::Superchargers, Element::Fire}},
-    {{3413, 0x0000}, {"Hurricane Jet Vac", Game::Superchargers, Element::Air}},
-    {{3413, 0x4503}, {"Legendary Hurricane Jet Vac", Game::Superchargers, Element::Air}},
-    {{3414, 0x0000}, {"Double Dare Trigger Happy", Game::Superchargers, Element::Tech}},
-    {{3414, 0x4502}, {"Power Blue Double Dare Trigger Happy", Game::Superchargers, Element::Tech}},
-    {{3415, 0x0000}, {"Super Shot Stealth Elf", Game::Superchargers, Element::Life}},
-    {{3415, 0x4502}, {"Dark Super Shot Stealth Elf", Game::Superchargers, Element::Life}},
-    {{3416, 0x0000}, {"Shark Shooter Terrafin", Game::Superchargers, Element::Earth}},
-    {{3417, 0x0000}, {"Bone Bash Roller Brawl", Game::Superchargers, Element::Undead}},
-    {{3417, 0x4503}, {"Legendary Bone Bash Roller Brawl", Game::Superchargers, Element::Undead}},
-    {{3420, 0x0000}, {"Big Bubble Pop Fizz", Game::Superchargers, Element::Magic}},
-    {{3420, 0x450E}, {"Birthday Bash Big Bubble Pop Fizz", Game::Superchargers, Element::Magic}},
-    {{3421, 0x0000}, {"Lava Lance Eruptor", Game::Superchargers, Element::Fire}},
-    {{3422, 0x0000}, {"Deep Dive Gill Grunt", Game::Superchargers, Element::Water}},
-    {{3423, 0x0000}, {"Turbo Charge Donkey Kong", Game::Superchargers, Element::Life}},
-    {{3423, 0x4502}, {"Dark Turbo Charge Donkey Kong", Game::Superchargers, Element::Life}},
-    {{3424, 0x0000}, {"Hammer Slam Bowser", Game::Superchargers, Element::Fire}},
-    {{3424, 0x4502}, {"Dark Hammer Slam Bowser", Game::Superchargers, Element::Fire}},
-    {{3425, 0x0000}, {"Dive-Clops", Game::Superchargers, Element::Water}},
-    {{3425, 0x450E}, {"Missile-Tow Dive-Clops", Game::Superchargers, Element::Water}},
-    {{3426, 0x0000}, {"Astroblast", Game::Superchargers}},
-    {{3426, 0x4503}, {"Legendary Astroblast", Game::Superchargers}},
-    {{3427, 0x0000}, {"Nightfall", Game::Superchargers}},
-    {{3428, 0x0000}, {"Thrillipede", Game::Superchargers, Element::Life}},
-    {{3428, 0x450D}, {"Eggcited Thrillipede", Game::Superchargers, Element::Life}},
-    {{3500, 0x0000}, {"Sky Trophy", Game::Superchargers}},
-    {{3501, 0x0000}, {"Land Trophy", Game::Superchargers}},
-    {{3502, 0x0000}, {"Sea Trophy", Game::Superchargers}},
-    {{3503, 0x0000}, {"Kaos Trophy", Game::Superchargers}}};
+    {{0, 0x0000}, {"Whirlwind", Game::SpyrosAdv, Element::Air, Type::Skylander}},
```

Comment:
I can see you've added the Type as an enum, did you have plans to use it for anything? Could potentially add another radio buttons component to the window, and put it here
<img width="836" alt="Screenshot 2023-08-11 at 9 03 02 AM" src="https://github.com/dolphin-emu/dolphin/assets/28370004/29d27ab3-71a1-4c54-a53c-88a063416063">

Reply:
Discussed in DM’s. Will add type filter similar to elements. Will add types for game gimmick figures (giants, swappers, trap masters, mini’s too probably). Type information was added to add seperate generation logic later down the line